src/converters.ts

```diff
@@ -80,7 +80,10 @@ export function attestationToVcFormat(payload: any): JwtCredentialPayload {
   return result
 }
 
-function normalizeJwtCredentialPayload(input: Partial<JwtCredentialPayload>): W3CCredential {
+function normalizeJwtCredentialPayload(
+  input: Partial<JwtCredentialPayload>,
+  removeOriginalFields: boolean = false
```

Comment:
I'm not sure that we need this behavior in the JWT->VC Case as well. I feel there's no use-case that requires to keep the JWT properties when extracting back the VC.

Rather I was even talking to Oliver that this function should probably throw errors if BOTH properties exist, BUT are not the same.

e.g.
input.sub -> result.credentialSubject.id

if both exist and are NOT equal -> throw -> invalid VC-JWT representation.

Reply:
Note, after reading the rest of the implementation: At the moment the implementations CAN produce JWT-VC with different values for there associated fields (e.g. `sub` and `credential.id`), however this should not be considered a valid credential and I still believe it should throw in this case.

Reply:
I created #64 to deal with this ambiguity.